doc/user/content/sql/prepare.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "PREPARE"
+description: "`PREPARE` creates a prepared statement (a server-side object that can be used to optimize performance)."
+menu:
+  main:
+    parent: "sql"
+---
+
+`PREPARE` creates a prepared statement. A prepared statement is a server-side object that can be used to optimize performance by separating planning from data.
+
+`PREPARE` parses, analyzes, and rewrites the initial `SELECT`, `INSERT`, `UPDATE`, or`DELETE` statement. A subsequent [`EXECUTE`] statement then plans and executes the statement. You can use also use `PREPARE` in conjunction with [`EXPLAIN`] to gain insight into a query's performance impact.
```

Comment:
space before DELETE

Reply:
I don't think I ever tested PREPARE/EXECUTE with EXPLAIN and I'm not sure what it does. It's definitely wasn't the intention of implementing these. I lean toward removing all EXPLAIN references in these changes unless there's a good reason to keep them.